### DISCUSSION TITLE
Support for pfSense Plus

### DISCUSSION THREAD

**Discussion by jaredhendrickson13** [jaredhendrickson13/pfsense-api#416] [Announcements]
A few weeks ago, [Netgate announced](https://www.netgate.com/blog/netgate-pfsense-plus-tac-lite-available-for-129-per-year) the discontinuation of free licensing for pfSense Plus for home and lab use. This license allowed for the API package to be easily included in automated testing within my existing lab environment and ultimately allowed me to comfortably support pfSense Plus. I want to iterate that I am still fully committed to supporting this package on pfSense Plus in any way can. However, there will be a few changes moving forward:

1. I am working on migrating my testing infrastructure for this project to AWS where I can continue to include pfSense Plus in automated testing without a significant financial impact. However, this also involves reworking many of the automated tests that are written today as they rely heavily on the current testing environment.
2. Starting with pfSense Plus 23.09, support for pfSense Plus will be labeled as 'community supported' for the remainder of API v1's life. This simply means I will continue to support pfSense Plus but the platform will no longer be included in the automated testing. I will heavily rely on the community to report issues relating to pfSense Plus compatibility going forward.
3. API v2 is currently in the works and making great progress. Tests for v2 will be written to use the aforementioned AWS testing infrastructure which will include pfSense Plus testing. I will make another announcement in the coming weeks with more details on v2.

Please help keep this project on healthy terms with the team at Netgate and refrain from including pfSense Plus source material in issues and PRs. 

Thanks for the support!

**Comment by philippenc** [comment]
Hello, I hope everything goes well for the future. I also hope that Netgate will include your work as an official package in future versions of pfsense.
Thank you for your work.
